### ISSUE TITLE
New Icon for FSearch

### ISSUE THREAD

**Issue by DroWnThePoor** [cboxdoerfer/fsearch#476]
I'm working on a video highlighting several apps for Linux, and FSearch is one of them.
Most of them have icons or branding, but FSearch on my system has a generic magnifying glass.
So I decided to make an icon, and I figured maybe it would be worth adding here to the project.
I've never contributed to a project aside from bug reports so forgive me if this is the wrong way to do this.

I've done a few iterations of the icon, and I can make any adjustments you might suggest. For instance, is it Fsearch or FSearch (like filesystem)? These are SVG's done in Inkscape.

Anyways here is an image of the 3 I have so far:
Also if you'd rather not use these, no problem at all I understand.

![image](https://user-images.githubusercontent.com/16339020/235415558-976f9d42-4709-42dd-a45e-3378401b30b5.png)

**Comment by cboxdoerfer** [comment]
Hi, wow thanks.

> Most of them have icons or branding, but FSearch on my system has a generic magnifying glass.

Yes, the current icon is as generic as it gets and was just added to have some kind of icon, as I'm not very familiar with Inkscape and designing icons myself.

> I've never contributed to a project aside from bug reports so forgive me if this is the wrong way to do this.

This way is perfectly fine. :+1: 

> For instance, is it Fsearch or FSearch (like filesystem)?

It's FSearch as in **F**ast,**F**ile or **F**ilesystem **Search**.

> Anyways here is an image of the 3 I have so far:
> Also if you'd rather not use these, no problem at all I understand.

Thank you! At first glance I like the first one the best and I think it has some potential.

Here are a few suggestions if you don't mind, but don't feel pressured to do any of that:

1. I'd prefer a less stylized font. Maybe you could use a simple sans serif font like Source Sans Pro or Inter instead.
2. You're using an ellipsis for the magnifying glass, probably to imply that it is tilted? I think highlighting its edges would improve this 3D effect. They could be displayed with a slightly darker tone (something like this: https://imgbin.com/png/iqj4hwH8/magnifying-glass-icon-science-icon-research-icon-png). Otherwise a using a circle instead might also work.
3. The reflection looks too realistic compared to the simplified magnifying glass. Something simpler like [this](https://www.iconsdb.com/white-icons/active-search-icon.html) or [this](https://www.crushpixel.com/de/stock-vector/search-icon-search-black-icon-1500833.html) might look better
4. The gradients in the folder icon also make it look too realistic for my taste compared to the magnifying glass. Solid colors (like [here](https://commons.wikimedia.org/wiki/File:OneDrive_Folder_Icon.svg) might look better
5. I think we could use more recognizable colors, instead of the more generic beige-yellowy folder color. I'll see if I can find a nice primary and secondary color.
6. Something to keep in mind is that the icon should still be recognizable at smaller sizes

Of course you're also free to use one of your icons for your video, if you don't want to wait until the official icon gets updated.

Thanks again!

**Comment by DroWnThePoor** [comment]
Okay so here are some revisions.
Sorry for the wait, time gets tight sometimes.
I may have gone too literally with it because I used the folder you shared, and I duplicated one of the magnifier glasses.
Font is Inter Medium, and added the drop shadowin
![image](https://user-images.githubusercontent.com/16339020/236995937-d531e417-9a56-48d1-91e3-49031311ea84.png)
g.

**Comment by nekohayo** [comment]
A problem I can foresee with these is that they probably won't work well at small sizes, whether on a dock/panel/dash (32-48px typically) or in menus (16, 22, 24 px), due to the amount of detail, and that they haven't been designed at those specific resolutions with pixels aligned to the grid on the canvas...

**Comment by DroWnThePoor** [comment]
That's correct, and he mentioned that above I believe.
This is just to get the general concept across it's not done at any particular scale.
I'm making a video highlighting several Linux apps and FSearch will be one of 10 or 12 apps in it, and many of them have non-generic icons. So I made this to make a non-generic one as well. It's intended for my dock which is 32px, but also at larger sizes. 
Because I've already done this I figured I could see if it would be usable to the project, and he made suggestions so I tried them. 
But this concept cannot work at those sizes. Those are usually more suited to symbolic icons no?

**Comment by Golddouble** [comment]
Thank you DroWnThePoor.

I like this symbol.
Do you have it as *.svg ?

If yes: Can you give us a link. I would like to have it for my panel.

**Comment by artytux** [comment]
@DroWnThePoor 
I agree the Icon is ok but could use some colour you have made some very good ones, Thank You

@nekohayo
I use  555 x 555 icons and never had any problems they fit they do as icons do, maybe KDE is more forgiving Heh heh.

@Golddouble I don't have any links though I made some icons for me from DroWnThePoor fine work, I will try to upload to here the sample and the folder with png and svg plus the xcf and text file in one folder, umm hope this works if not --

![0704,1001 Sample of icons in folder](https://github.com/cboxdoerfer/fsearch/assets/111945598/7462315c-0d5b-48b3-9d73-13926a914abb)

[FSearch Icons__png and svg and the xcf for The Gimp.zip](https://github.com/cboxdoerfer/fsearch/files/11943370/FSearch.Icons__png.and.svg.and.the.xcf.for.The.Gimp.zip)

OK had to compress as zip seems to have worked Ok
The two items at bottom of sample sheet are text file and the Gimp xcf canvas file if/for anyone who want to experiment/play  with the images.


Regards
artytux

**Comment by artytux** [comment]
Should have mentioned that the xcf Gimp image file has the different parts sections as separate layers in a few colours
you could have some parts as transparent by clicking the eye icon off,
there is also alpha locked on some layers disable by clicking the alpha lock button that was for brushing different colours on,
opacity is changeable also
and experiment to your hearts contentment.

Regards
artytux

**Comment by Golddouble** [comment]
Thank you artytux. I will use the icon with the blue background and the black loop.

(But actually I was asking for the icon on the thread from May 9. (No background and "FS" instead of "FSearch".)

**Comment by artytux** [comment]
@Golddouble Thanks for the feedback the icon I used is the great graphics from @DroWnThePoor the middle one on May 1 post,

I work from home and that can be in itself difficult with time management,
I should have another lot of icons with and without backgrounds will try to include short and long handles also FSearch and "FS" and as I"m writing this I see a few simple ideas, I will keep the branding simpler and easier to follow,

some days too many thing are going on all at near same time period and it's a bit much,
and I forget to walk away from the work to go outside and have a break.
I'd like to give a reasonable selection will also include The Gimp xcf image file with all the layers included,
so others can have a bit of fun.

Should be ready about Mon or Tuesday

@DroWnThePoor apologizes are in order, I have hijacked your thread, you got great artwork, I could not stop myself.

Regards
artytux

**Comment by Golddouble** [comment]
Thank you. No problem. There is no hurry.

**Comment by artytux** [comment]
@Golddouble 
> But actually I was asking for the icon on the thread from May 9. (No background and "FS" instead of "FSearch".

I was curious and interested to see the results :-)

FSearch Icon -_- Folder and FSearch on Alpha.png








FSearch Icon -_- Folder and FS on Alpha.png

**Comment by artytux** [comment]
@Golddouble 
It's on the list,  just a few more acrylic on canvas painting jobs orders received got ordered to be made, very unexpected very nice though more work equals bit more money.

I will get to the yellow svg icon (plus extras haven't forgot) this weekend also will add an updated Gimp xcf file Inc a read me txt with a few instructions about
The Gimp (create the png ) and LazPaint convert to svg format,
So no others wants ? .
Easily pleased around here , sweet .

To admin a Question
The xcf file could end up a big heavy hopefully can upload or would it be better to split the xcf file be into smaller files ? (zips) . 

Regards
artytux

**Comment by artytux** [comment]
@Golddouble
Since no one asked for anything, had a opportunity to work on yours
xcf file with the icon split into 8 parts and 8 colours and fonts yeah right so many so good.
I going to make few different colours in the xcf so then I will see what I like best, if you want I'll post a previews selection.
the grey colour is my window background, the icons for you have alpha background.

Here the png made with Gimp and converted to svg with LazPaint

FSearch Icon-1bb_Black FS and Black Loop.png
![FSearch Icon-1bb_Black FS and Black Loop](https://github.com/cboxdoerfer/fsearch/assets/111945598/56bba714-f12a-4c9f-96f2-5422513b0db5)

FSearch Icon-1bb_Black FS and Black Loop.svg
![FSearch Icon-1bb_Black FS and Black Loop](https://github.com/cboxdoerfer/fsearch/assets/111945598/a70e2b08-81ec-408a-a74b-2b811798f78d)

Regards
artytux

**Comment by Golddouble** [comment]
This is very cool. Thank you. I like it.

**Comment by artytux** [comment]
> This is very cool. Thank you. I like it.


I'm glad that you like the icon (phew), most welcome @Golddouble  and Thank You for your Thank You.

regards
artytux
